### PR TITLE
Fixed extraVolumeMounts bug #1824

### PR DESCRIPTION
**Pull request by LockedThread**
…c7eb8a637

Fixes extraVolumeMounts, read issue https://github.com/apollographql/router/issues/1824 for more info.

### CODE REVIEW

**Comment by LockedThread** [comment]
adding changelog

**Review by garypen — COMMENTED** [review]
Good catch. Sorry about the regression. We've improved our release process, so you don't need to update:
 - Chart.yaml
 - README.md

(we do that automatically when we cut a release).

I'll revert those files and then approvethe fix.

File: helm/chart/router/Chart.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.0.0-rc.1
+version: 1.0.0-rc.2
```
Comment (abernix) [inline helm/chart/router/Chart.yaml:18]:
```suggestion
version: 1.0.0-rc.1
```

File: helm/chart/router/README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 [router](https://github.com/apollographql/router) Rust Graph Routing runtime for Apollo Federation
 
-![Version: 1.0.0-rc.1](https://img.shields.io/badge/Version-1.0.0--rc.1-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v1.0.0-rc.1](https://img.shields.io/badge/AppVersion-v1.0.0--rc.1-informational?style=flat-square)
+![Version: 1.0.0-rc.2](https://img.shields.io/badge/Version-1.0.0--rc.2-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v1.0.0-rc.2](https://img.shields.io/badge/AppVersion-v1.0.0--rc.2-informational?style=flat-square)
```
Comment (abernix) [inline helm/chart/router/README.md:5]:
```suggestion
![Version: 1.0.0-rc.1](https://img.shields.io/badge/Version-1.0.0--rc.1-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v1.0.0-rc.1](https://img.shields.io/badge/AppVersion-v1.0.0--rc.1-informational?style=flat-square)
```

File: helm/chart/router/README.md [hunk 3]
```diff
@@ -11,15 +11,15 @@
 ## Get Repo Info
 
 ```console
-helm pull oci://ghcr.io/apollographql/helm-charts/router --version 1.0.0-rc.1
+helm pull oci://ghcr.io/apollographql/helm-charts/router --version 1.0.0-rc.2
```
Comment (abernix) [inline helm/chart/router/README.md:14]:
```suggestion
helm pull oci://ghcr.io/apollographql/helm-charts/router --version 1.0.0-rc.1
```

File: helm/chart/router/README.md [hunk 4]
```diff
@@ -11,15 +11,15 @@
...
 ```
 
 ## Install Chart
 
 **Important:** only helm3 is supported
 
 ```console
-helm upgrade --install [RELEASE_NAME] oci://ghcr.io/apollographql/helm-charts/router --version 1.0.0-rc.1 --values my-values.yaml
+helm upgrade --install [RELEASE_NAME] oci://ghcr.io/apollographql/helm-charts/router --version 1.0.0-rc.2 --values my-values.yaml
```
Comment (abernix) [inline helm/chart/router/README.md:22]:
```suggestion
helm upgrade --install [RELEASE_NAME] oci://ghcr.io/apollographql/helm-charts/router --version 1.0.0-rc.1 --values my-values.yaml
```

**Review by abernix — COMMENTED** [review]
We should probably make the instructions in the `Chart.yaml` more clear so users know they don't need to bump this (or even run `helm docs`.)

**Comment by garypen** [comment]
Ok. I couldn't update your PR, so I created a replacement and marked you as the co-author with a note that you were the sole author. Closing this as it is replaced by #1831 .

**Comment by LockedThread** [comment]
> Good catch. Sorry about the regression. We've improved our release process, so you don't need to update:
> 
>  - Chart.yaml
> 
>  - README.md
> 
> 
> 
> (we do that automatically when we cut a release).
> 
> 
> 
> I'll revert those files and then approvethe fix.

Oh okay thats good I haven't worked on the helm chart since my previous changes a couple months ago when you had to. Thanks for letting me know, I'll be coordinating with you for the other PR I have open.

**Comment by LockedThread** [comment]
> We should probably make the instructions in the `Chart.yaml` more clear so users know they don't need to bump this (or even run `helm docs`.)

Definitely would be a good idea.

**Pull request closed without merging**